gulp-tasks/dist-darwin.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 const download = require('./download.js');
-const reqs = require('../requirements-darwin.json');
+const reqs = require('../requirements.json');
```

Comment:
Any particular reason the reqs are loaded in a different fashion compared to windows?